kebechet/managers/update/update.py

```diff
@@ -101,6 +103,9 @@ def _get_dependency_version(dependency: str, is_dev: bool) -> str:
         normalized_dependency = re.sub(r"[-_.]+", "-", dependency).lower()
         version = pipfile_lock_content['develop' if is_dev else 'default'].get(
             normalized_dependency, {}).get('version')
+        # if not kept as normaized depedency in Pipfile.lock.
+        version = pipfile_lock_content['develop' if is_dev else 'default'].get(
```

Comment:
Please check, added as it was having problems with "delegator.py" - 
https://github.com/saisankargochhayat/kebechet_sample/blob/6c51e54b6c7f684573c8752b80aebdf142292f1d/Pipfile.lock#L115

Reply:
is there a requirement to use `normalized_dependency`?

Reply:
https://github.com/thoth-station/kebechet/issues/209